README.md

```diff
@@ -185,11 +185,9 @@ Prebuilt binaries are distributed via `npm`. Run `npx workerd ...` to use these.
 
 ### Local Worker development with `wrangler`
 
-[Wrangler](https://developers.cloudflare.com/workers/wrangler/) has experimental support for running Workers with `workerd`:
+You can use [Wrangler](https://developers.cloudflare.com/workers/wrangler/) to develop Cloudflare Workers locally, using `workerd`. Run:
 
-`wrangler dev --experimental-local`
-
-This feature is under active development.
+`wrangler dev --local`
```

Comment:
```suggestion
`wrangler dev`
```

The `--local` flag currently warns you that it is no-longer needed and will be removed in a future version